spaces/domains.json

```
@@ -137,6 +137,7 @@
   "dao.zfo.gg": "zfogg.eth",
   "vote.ensuser.com": "ensuser.eth",
   "governance.diamondhandsfinance.org": "diamondhandsfinance.eth",
+  "vote.bears.finance":"fbear.eth",
```

Could you just add a space here please? Thank you 😊 
```suggestion
  "vote.bears.finance": "fbear.eth",
```

---

done